xml/slurm.xml

```diff
@@ -49,7 +49,7 @@
   <sect2 xml:id="sec-slurm-minimal">
    <title>Minimal installation</title>
    <important>
-    <title>Make Sure of Consistent UIDs and GIDs for Slurm's Accounts</title>
+    <title>Make sure of consistent UIDs and GIDs for slurm's accounts</title>
```

Comment:
Isn't Slurm a product name here? (And Futurama reference, of course.)

Reply:
```suggestion
    <title>Make sure of consistent UIDs and GIDs for Slurm's accounts</title> 
```